api/observability/tracing/ssh/channel.go

```diff
@@ -59,15 +58,13 @@ func (c *Channel) SendRequest(ctx context.Context, name string, wantReply bool,
 			semconv.RPCSystemKey.String("ssh"),
 		),
 	)
-	defer span.End()
+	defer func() {
+		tracing.EndSpan(span, err)
+	}()
```

Comment:
bikeshedding: single line?

```suggestion
	defer func() { apitracing.EndSpan(span, err) }()
```